example/lib/main.dart

```diff
@@ -1 +1,7 @@
-void main() {}
+void info() {
```

Comment:
<table>
  <tbody>
    <tr>
  <td>:speech_balloon:</td>
  <td>The value of the local variable 'unused' isn't used.</td>
</tr>
  </tbody>
</table>